src/compiler.ts

```diff
@@ -1480,11 +1516,11 @@ export class Compiler extends DiagnosticEmitter {
     contextualTypeArguments: Map<string,Type> | null = null,
     alternativeReportNode: Node | null = null
   ): void {
-    // TODO
-    this.error(
-      DiagnosticCode.Not_implemented,
-      declaration.range
-    );
+    // TODO Compile functions to use
+    // this.error(
+    //   DiagnosticCode.Operation_not_supported,
+    //   declaration.range
+    // );
```

Comment:
Is there missing something here?

Reply:
Ah this could be used to merge multiple interface declarations.